.github/workflows/test-forked.yml

```
@@ -21,6 +21,11 @@ jobs:
     with:
       forked: true
 
+  check-licenses:
+    uses: ./.github/workflows/check-licenses.yml
+    with:
+      forked: false
```

I think this should be true
```suggestion
      forked: true
```

---

agreed
```suggestion
      forked: true
```